### PR TITLE
Focus 195

### PR DESCRIPTION
**Pull request by ArushiSharma95**
FOCUS-195: Schedule widget component

### CODE REVIEW

File: frontend/src/features/schedule/ScheduleList.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import { React, useState, useEffect } from 'react';
+import { Providers, ProviderState } from '@microsoft/mgt-element';
+import { Agenda } from '@microsoft/mgt-react';
+
+//helps in tracking user's sign in
+function useIsSignedIn() {
```
Comment (jsnkle) [inline frontend/src/features/schedule/ScheduleList.js:6]: Should this be userIsSignedIn ?

**Review by jsnkle — COMMENTED** [review]
Just had the one question...

File: frontend/src/features/schedule/ScheduleList.js (hunk 1 above)
Comment (ArushiSharma95) [reply frontend/src/features/schedule/ScheduleList.js:6]: No, it should be starting with "use".
Do you think I should be changing it to useIsUserSignedIn?

Comment (jsnkle) [reply frontend/src/features/schedule/ScheduleList.js:6]: Ohhhh... sorry was reading that wrong. No, what  you have is good.

**Review by jsnkle — APPROVED** [review]

File: frontend/src/index.js [hunk 2]
```diff
@@ -6,6 +6,14 @@ import App from "app/App";
 import { store } from "app/store";
 import { Provider } from "react-redux";
 import reportWebVitals from "./reportWebVitals";
+import { Providers } from '@microsoft/mgt-element';
+import { Msal2Provider } from '@microsoft/mgt-msal2-provider';
+
+Providers.globalProvider = new Msal2Provider({
+	clientId: 'dd6e9c0d-c37c-4a0e-969e-0538656f58b6',
```
Comment (rbadamsjr) [inline frontend/src/index.js:13]: Are we moving the clientId to a environment var?

**Review by rbadamsjr — APPROVED** [review]
Thanks for answering the questions.

**Pull request merged**